azkaban-web-server/src/web/js/azkaban/view/flow-trigger-list.js

```diff
@@ -229,11 +232,15 @@ azkaban.FlowTriggerInstanceListView = Backbone.View.extend({
     var startTime = node.dependencyStartTime == 0 ? (new Date()).getTime()
         : node.dependencyStartTime;
 
-    var endTime = node.dependencyEndTime == 0 ? (new Date()).getTime()
-        : node.dependencyEndTime;
+    var endTime = node.dependencyEndTime;
 
     $(tdStart).text(getDateFormat(new Date(startTime)));
-    $(tdEnd).text(getDateFormat(new Date(endTime)));
+    if (node.dependencyEndTime == 0) {
```

Comment:
Maybe combine the if-else statement of `tdEnd` and `tdElapse`? Also do you need that `var endTime = node.dependencyEndTime;`?

Reply:
updated, thanks